gcloud/_helpers.py

```diff
@@ -409,15 +409,23 @@ def _rfc3339_nanos_to_datetime(dt_str):
     return bare_seconds.replace(microsecond=micros, tzinfo=UTC)
 
 
-def _datetime_to_rfc3339(value):
-    """Convert a native timestamp to a string.
+def _datetime_to_rfc3339(value, ignore_zone=True):
+    """Convert a timestamp to a string.
 
     :type value: :class:`datetime.datetime`
     :param value: The datetime object to be converted to a string.
 
+    :type ignore_zone: boolean
+    :param ignore_zone: If True, then the timezone (if any) of the datetime
+                        object is ignored.
+
     :rtype: str
     :returns: The string representing the datetime stamp.
     """
+    if not(ignore_zone or value.tzinfo is None):
```

Comment:
Can you put a space after `not` or use DeMorgan's law and say

```python
if not ignore_zone and value.tzinfo is not None:
    ...
```

I prefer the 2nd. I had to ponder on this boolean for a few extra moments to figure out what it does (if we don't ignore the zone and there is a zone there, then we remove the offset)